### PR TITLE
fix(jira-status): #CO-1780 update jira status when update ticket

### PR DESCRIPTION
**Pull request by Ozen40**
## Describe your changes
When you update a ticket, it will re-display the escalation status
## Issue ticket number and link
[CO-1780](https://jira.support-ent.fr/browse/CO-1780)
## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

**Review by simonchaval — APPROVED** [review]

File: src/main/resources/public/ts/controllers/controller.ts [hunk 1]
```diff
@@ -616,6 +616,10 @@ export const SupportController: Controller = ng.controller('SupportController',
 
 			});
 
+			if ($scope.ticket.escalation_status != 1) {
```
Comment (qpic-cgi) [inline src/main/resources/public/ts/controllers/controller.ts:619]: c'est quoi le escalation_status 1 ?

File: src/main/resources/public/ts/services/ticket.service.ts [hunk 2]
```diff
@@ -44,6 +45,16 @@ export const ticketService: ITicketService = {
...
-            .then((res: AxiosResponse) => (<ICanAccessResponse>res.data).canAccess)
+            .then((res: AxiosResponse) => (<ICanAccessResponse>res.data).canAccess),
+
+    /**
+     * check if the structure and the user have a certain workflow
+     *
+     * @param ticketJiraId {string} id of the ticket in Jira
+     * @returns {void}
+     **/
+    updateJiraStatus(ticketJiraId: string) {
```
Comment (qpic-cgi) [inline src/main/resources/public/ts/services/ticket.service.ts:56]: typage

File: src/main/resources/public/ts/controllers/controller.ts [hunk 3]
```diff
@@ -616,6 +616,10 @@ export const SupportController: Controller = ng.controller('SupportController',
...
+				ticketService.updateJiraStatus($scope.ticket.issue.id_jira);
+			}
```
Comment (lapeyronnie) [inline src/main/resources/public/ts/controllers/controller.ts:621]: Il n'y a pas de gestion de l'erreur

**Review by lapeyronnie — CHANGES_REQUESTED** [review]

**Review by qpic-cgi — APPROVED** [review]

**Review by qpic-cgi — CHANGES_REQUESTED** [review]
fail, j'approuve pas encore ^^

File: src/main/resources/public/ts/controllers/controller.ts (hunk 1 above)
Comment (qpic-cgi) [reply src/main/resources/public/ts/controllers/controller.ts:619]: (en lisant le code, je ne suis pas sensé devoir deviner ce que tu as écrit, tu devrais donc pour tes escalation_status utiliser un enum, afin que le "slug" du status 1, me donne une indication à la lecture du code. Exemple:

enum => OPEN(1)

if($scope.ticket.escalation_status != enum.OPEN)
)

**Pull request closed without merging**